Formula/cloud-nuke.rb

```diff
@@ -0,0 +1,26 @@
+class CloudNuke < Formula
+  desc "Clean up your cloud accounts by nuking all resources in it"
+  homepage "https://gruntwork.io/"
+  url "https://github.com/gruntwork-io/cloud-nuke/archive/v0.1.4.tar.gz"
+  sha256 "ce2f5caff4db80accc9ceaeef859382135e772dfd08d1bf981e52098512c45f7"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    print buildpath
+    ENV["GOPATH"] = buildpath
+    cloud_nuke_path = buildpath/"src/github.com/gruntwork-io/cloud-nuke/"
+    cloud_nuke_path.install buildpath.children
+
+    cd cloud_nuke_path do
+      system "dep", "ensure", "-vendor-only"
+      system "go", "build", "-ldflags", "-X main.VERSION=#{version}"
+      bin.install "cloud-nuke"
+    end
+  end
+
+  test do
+    assert_match "cloud-nuke version 0.1.4", shell_output("#{bin}/cloud-nuke -v ")
```

Comment:
I understand what you are requesting. I think it will be quite difficult to mock out an AWS account to clear the resources inside. Can you suggest a better test case? I will be more than happy to implement it.

Reply:
hi @fxcoudert  can you elaborate please

Reply:
I think it could be the same as https://github.com/Homebrew/homebrew-core/pull/34429
The reasoning here is to check that build is successful, being version usually don't need any dependencies etc. Making request we at least know that dependency for API/network correct and there are no segfaults etc. Not necessarily about this formula but as a general rule. You can read more about it [here](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula).